doc/maintainers.rst

```diff
@@ -1,49 +1,52 @@
+:orphan:
```

Comment:
Did sphinx complained this not being in any toctree? I missed that warning. But this makes me think if we should put this on our docs. Kind of reduce cognitive burden. All docs are online, but the release instructions are only in the repo. 
Also why all the new white space?

Reply:
> Did sphinx complained this not being in any toctree? 

Yes I was getting rid of warnings

> Also why all the new white space?

Getting rid of sphinx warnings about bad spacing (which are in `master`). But really these should all be part of the same bulleting indentation so it's a bugfix of sorts, too.

Reply:
We can add it to the docs later if someone is motivated, not sure there is much to gain (users don't care and maintainers can just look in the repo)